doc/policies_list/3.10.0/policies.json

```diff
@@ -0,0 +1,2718 @@
+                "properties": {
+                  "url": {
+                    "type": "string",
+                    "description": "New URL in case of match"
+                  },
+                  "regex": {
+                    "type": "string",
+                    "description": "Regular expression to be matched"
+                  }
+                },
+                "required": [
+                  "regex",
+                  "url"
+                ],
+                "type": "object"
+              },
+              "description": "List of rules to be applied",
+              "type": "array"
+            }
+          },
+          "type": "object"
+        },
+        "version": "builtin"
+      }
+    ],
+    "rewrite_url_captures": [
+      {
+        "summary": "Captures arguments in a URL and rewrites the URL using them.",
+        "description": [
+          "Captures arguments in a URL and rewrites the URL using these arguments. ",
+          "For example, we can specify a matching rule with arguments like ",
+          "'/{orderId}/{accountId}' and a template that specifies how to rewrite ",
+          "the URL using those arguments. Example: ",
+          "'/sales/v2/{orderId}?account={accountId}'; in this case, the request ",
+          "'/123/456' will be transformed into '/sales/v2/123?account=456'"
+        ],
+        "name": "URL Rewriting with Captures",
+        "$schema": "http://apicast.io/policy-v1.1/schema#manifest#",
+        "configuration": {
+          "definitions": {
+            "methods": {
+              "items": {
+                "oneOf": [
+                  {
+                    "title": "GET",
+                    "enum": [
+                      "GET"
+                    ]
+                  },
+                  {
+                    "title": "POST",
+                    "enum": [
+                      "POST"
+                    ]
+                  },
+                  {
+                    "title": "PUT",
+                    "enum": [
+                      "PUT"
+                    ]
+                  },
+                  {
+                    "title": "PATCH",
+                    "enum": [
+                      "PATCH"
+                    ]
+                  },
+                  {
+                    "title": "DELETE",
+                    "enum": [
+                      "DELETE"
+                    ]
+                  },
+                  {
+                    "title": "HEAD",
+                    "enum": [
+                      "HEAD"
+                    ]
+                  },
+                  {
+                    "title": "OPTIONS",
+                    "enum": [
+                      "OPTIONS"
+                    ]
+                  }
+                ],
+                "type": "string"
+              },
+              "description": "Array of HTTP methods this rule must be applied to. If left blank it will be applied to all HTTP methods",
```

Comment:
```suggestion
              "description": "An array of HTTP methods this rule must be applied to. If left blank it will be applied to all HTTP methods.",
```